UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs

```diff
@@ -342,7 +342,7 @@ private static string StripAll(string input)
 		return output;
 	}
 
-	private static string StripTags(string input)
+	public static string StripTags(string input)
 	{
 		//Regex - find "<" followed by any number of not ">" and ending in ">". Matches any HTML tags.
 		Regex rx = new Regex("[<][^>]+[>]");
```

Comment:
just saying this stuff is pretty slow when I did profile it

Reply:
The only ones server side i added were the ones for the comms console. The chat bubble will strip client side

Reply:
I'm not sure if theres a better way to do it?

Reply:
I think there is built-in replace that might work well <  with ""

Reply:
It also gets rid of the stuff in between the <>

Reply:
yeah, it is stripping the entire tag so I think regex is needed here. Another way would be finding the "<" character taking note of its index then find the ">" character and make a substring but that's is way too prone to fail in my opinion.

Reply:
but it's fine currently